reflect/src/main/java/dagger/reflect/ReflectiveModuleParser.java

```diff
@@ -1,13 +1,6 @@
 package dagger.reflect;
 
-import static dagger.reflect.Reflection.boxIfNecessary;
-import static dagger.reflect.Reflection.findAnnotation;
-import static dagger.reflect.Reflection.findMapKey;
-import static dagger.reflect.Reflection.findQualifier;
-import static dagger.reflect.Reflection.findScope;
-import static dagger.reflect.Reflection.findScopes;
-import static dagger.reflect.Reflection.maybeInstantiate;
-import static dagger.reflect.Reflection.requireAnnotation;
+import static dagger.reflect.Reflection.*;
```

Comment:
wildcard imports are not allowed in google java format. i'm surprised this passed verification.

Reply:
Yeah, weird that `./gradlew googleJavaFormat` was fine with it. Reverted the change.